### ISSUE TITLE
apoc.cypher.runFirstColumn

### ISSUE THREAD

**Issue by DevJoseWeb** [grand-stack/grand-stack-starter#10]
Did you help with this error?
I'm fascinated by this Nietzsche combination, a powerful technology bomb. I'm trying to upload everything to see how it works and do this project here in GRANDstack.
https://github.com/DevJoseWeb/SIMPLUS
video that I recorded with the error:
https://www.youtube.com/watch?v=QOZJZ1Rft1c&feature=youtu.be
my java project that I'm going to migrate to GRANDstack
https://www.youtube.com/watch?v=EBKLgGdJ-LA
my erro:
[GraphQL error]: Message: Failed to invoke function `apoc.cypher.runFirstColumn`: Caused by: java.lang.IllegalArgumentException: Can not be converted to long: org.neo4j.kernel.impl.core.NodeProxy, Location: [object Object], Path: users
service-worker.js:1 Uncaught (in promise) Error: Request for https://grand-stack-starter-ui-fycyjljinn.now.sh/index.html?_sw-precache=8f144f72c04160c31b6abb5e5553f4db returned a response with status 301
    at service-worker.js:1
(anonymous) @ service-worker.js:1

**Comment by MistaTwista** [comment]
If you still not solve this problem you should try to [install APOC](https://github.com/neo4j-contrib/neo4j-apoc-procedures#apoc-name-history) in your Neo4j instance. I've installed it in Neo4j Desktop and it works

**Comment by DevJoseWeb** [comment]
I installed as you taught and it worked, thank you! What project are you working on at the moment?

**Comment by johnymontana** [comment]
Right, the APOC plugin for Neo4j is required when using the `@cypher` directive. We've updated the README to make this a bit more clear [here](https://github.com/grand-stack/grand-stack-starter#neo4j)

**Comment by DevJoseWeb** [comment]
johnymontana
MistaTwista
Tanks Mans !
